test-os/min_clock_os.ml

```diff
@@ -4,7 +4,11 @@
    ocamlfind ocamlc \
      -package ptime.clock.os -linkpkg -o min_clock_os.byte min_clock_os.ml
    ocamlfind ocamlopt \
-     -package ptime.clock.os -linkpkg -o min_clock_os.native min_clock_os.ml *)
+     -package ptime.clock.os -linkpkg -o min_clock_os.native min_clock_os.ml
+   js_of_ocaml \
+     $(ocamlfind query ptime.clock.os -predicates javascript -o-format -r | xargs echo) \
```

Comment:
We could add a `-package` flag to js_of_ocaml. It already knows about ocamlfind (optional deps)  to resolve the  `+pkg/file.js`

Reply:
This or say something about it in the docs of `js_of_ocaml` we need to know how `js_of_ocaml` is supposed to be used with the eco-system. Use `dune` is not a suitable answer as far as I'm concerned.